tests/integration/common/auth/test_auth.py

```diff
@@ -101,3 +104,24 @@ def test_post_claims_link_and_redirects(self, client, factories):
         assert response.status_code == 302
         assert response.location == "/my-redirect"
         assert magic_link.claimed_at_utc is not None
+        assert current_user.is_authenticated is True
+        assert magic_link.user.id == current_user.id
+
+
+class TestSignOutView:
+    def test_get(self, client, factories):
+        magic_link = factories.magic_link.create(
+            user__email="test@communities.gov.uk", redirect_to_path="/my-redirect", claimed_at_utc=None
+        )
+
+        # A bit unencapsulated for testing the sign out view, but don't otherwise have an easy+reliable way to get
+        # the user in the session
+        client.post(url_for("auth.claim_magic_link", magic_link_code=magic_link.code), json={"submit": "yes"})
+        with client.session_transaction() as session:
+            assert "_user_id" in session
+
+        response = client.get(url_for("auth.sign_out"), follow_redirects=True)
+        assert response.status_code == 200
+
+        with client.session_transaction() as session:
+            assert "_user_id" not in session
```

Comment:
Nice 👍 feels OK to do this setting it up to claim it initially in the context of magic links + sessions (although I'm sure future tests will want a user fixture managed on the session somewhere)